### PR TITLE
[ADD] survey_percent_question

### PR DESCRIPTION
**Pull request by veloutin**
This module adds a new question type "Percent" which allows splitting a percentage value across
a list of options.

### CODE REVIEW

File: survey_percent_question/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* For questions, please contact our support services <support@savoirfairelinux.com>
+
+Maintainer
+==========
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (Ehtaga) [inline survey_percent_question/README.rst:46]: Just some nitpicking: there is a more recent Readme OCA template,  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: survey_percent_question/models/survey.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            a_tag = "{0}_{1}".format(answer_tag, row.id)
+            self.create(dict(defaults, **{
+                "value_number": float(to_decimal(answers.get(a_tag))),
+                "value_suggested_row": row.id,
+            }))
+
+        return True
+
+
+class SurveyQuestion(models.Model):
```
Comment (Ehtaga) [inline survey_percent_question/models/survey.py]: You should put each different class in a separate model file.

**Comment by Ehtaga** [comment]
Looks good to me :+1:

File: survey_percent_question/README.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
+Features
```
Comment (pedrobaeza) [inline survey_percent_question/README.rst]: Put module title here

File: survey_percent_question/README.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+========
+
+This module was written to offert a new type of question. 
+You can now ask in a survey the repartition in % though a chosen number of option 
+
+Installation
+============
+
+To install this module, you just need to select the module and insure yourself dependancies are available.
```
Comment (pedrobaeza) [inline survey_percent_question/README.rst]: As dependencies are from core, this section is not needed

File: survey_percent_question/README.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
+Features
+========
+
+This module was written to offert a new type of question.
```
Comment (pedrobaeza) [inline survey_percent_question/README.rst]: s/offert/offer

File: survey_percent_question/README.rst [hunk 6]
```diff
@@ -0,0 +1,45 @@
+Features
+========
+
+This module was written to offert a new type of question. 
+You can now ask in a survey the repartition in % though a chosen number of option
```
Comment (pedrobaeza) [inline survey_percent_question/README.rst]: s/option/options

**Comment by pedrobaeza** [comment]
You should split the module in 2: one for the backend, and another for the website part.

**Comment by rafaelbn** [comment]
Please @JavierIniesta @JMODaERP review, thanks!

**Comment by rafaelbn** [comment]
> You should split the module in 2: one for the backend, and another for the website part.

Hi @veloutin @Ehtaga did you split module as @pedrobaeza said?

Thanks a lot for you contributions and sorry in this big delay.

**Comment by gurneyalex** [comment]
@pedrobaeza what's the benefit of splitting?

**Comment by pedrobaeza** [comment]
Don't depend on frontend part if you only want percentage on backend surveys (for example, for internal HR evaluation)

**Comment by rafaelbn** [comment]
Hi @Ehtaga @veloutin I'm sorry about this PR. I would like to review it and go ahead if you are interested. Are you?
@pedrobaeza survey module depends on website, so is not possible to work only with backend.
I'm rebuilding runbot for testing

**Comment by rafaelbn** [comment]
Tested in runbot :+1:

**Comment by rafaelbn** [comment]
Ready to be merged

**Pull request merged**